scripts/index.js

```diff
@@ -0,0 +1,212 @@
+import { formUser } from './modules/form.js';
+import { News } from './modules/news.js';
+
+News();
```

Comment:
Not sure if we use News and formUser. I suppose both could be removed.

Reply:
Fixed.
Fixes deleted